project-base/app/tests/FrontendApiBundle/Functional/Price/HiddenPricesTest.php

```diff
@@ -99,4 +100,30 @@ public function testCustomerUserCannotSeeOrderPrices(): void
             $this->assertSame('***', $item['unitPrice']['priceWithoutVat']);
         }
     }
+
+    public function testCustomerUserCannotSeeGatewayPayments(): void
+    {
+        $query = '
+            query {
+                payments {
+                    uuid
+                    type
+                    price {
+                        priceWithVat
+                        priceWithoutVat
+                    }
+                }
+            }
```

Comment:
could you please define the query in a separate graphql file? :innocent:

Reply:
Do you guys define separate graphql file  even for small queries ? Therefore I have to create separate query files for previous tests. Am I right ?

Reply:
Fixed in https://github.com/shopsys/shopsys/pull/3356/commits/94767ba63cb33dd7289ccdb0db9531ff00c3cd22